app/core/schools/schools_type.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+
+
+class SchoolType(str, Enum):
+    """
+    In Hyperion, each user must have a school. Belonging to a school gives access to a set of specific endpoints.
```

Comment:
The name SchoolType and what exactly is does is confusing, could you add more comments ?
(for ex what about students that are neither `no_school` and `centrale_lyon`)